packages/console-handler/src/consoleFormatter.ts

```diff
@@ -1,23 +1,39 @@
 import { dim, gray, yellow, red, white, blue, unstyle } from 'ansi-colors';
-import { FormatterPlugin, PluginManager } from '../plugins';
-import { LogEntry } from '../queues';
-import { Formatter } from './formatter';
-import { formatData, isEmpty } from './utils';
+import {
+  FormatterPlugin,
+  PluginManager,
+  LogEntry,
+  Formatter,
+  formatData,
+  isEmpty,
+  TContextBase,
+  TContextShape,
+} from '@debugr/core';
 
-export class ConsoleFormatter extends Formatter {
+export class ConsoleFormatter<
+  TTaskContext extends TContextBase,
+  TGlobalContext extends TContextShape,
+> extends Formatter<Partial<TTaskContext>, TGlobalContext> {
   private readonly writeTimestamp: boolean;
 
-  constructor(pluginManager: PluginManager, writeTimestamp: boolean = true) {
+  constructor(
+    pluginManager: PluginManager<Partial<TTaskContext>, TGlobalContext>,
+    writeTimestamp: boolean = true,
+  ) {
     super(pluginManager);
     this.writeTimestamp = writeTimestamp;
   }
 
-  protected formatEntry(entry: LogEntry, previousTs?: number, plugin?: FormatterPlugin): string {
+  protected formatEntry(
+    entry: LogEntry<Partial<TTaskContext>, TGlobalContext>,
+    _previousTs?: Date,
```

Comment:
proč je tam to podtržítko?

Reply:
unused parameter

Reply:
v konvenci, co znam, se podtrzitkem prefixne unused parameter